manifests/mod/security.pp

```diff
@@ -32,6 +32,9 @@
 # 
 # @param audit_log_type
 #   Defines the type of audit logging mechanism to be used.
+#
+# @param audit_log_format
+#   Defines what format the logs should be written in. Accepts `Native` and `JSON`.
```

Comment:
No need to list accepted values, as the documentation include the data type (and the default value).